lib/hammer_cli_katello/host_extensions.rb

```diff
@@ -57,18 +58,28 @@ module HostExtensions
                   _("Content View Environment Labels"), Fields::Field
             collection :content_view_environments, _('Content View Environments') do
               from :content_view do
-                label _("Content view") do
-                  field :id, _("Id")
-                  field :name, _("Name")
-                  field :composite, _("Composite"), Fields::Boolean
-                  field :rolling, _("Rolling"), Fields::Boolean
+                # Deprecated label. To be removed in future versions.
+                label _("Content view"), :sets => ['ALL'] do
+                  field :id, _("Id"), Fields::Field, :replaced_by => "Content Information/Content View Environments/CV Id"
```

Comment:
```suggestion
                  field :id, _("Id"), Fields::Field, :replaced_by => _("Content Information/Content View Environments/CV Id")
```

Same for the other replacements. It should do the trick, I hope.

Reply:
Thank you for the review! I've just created a new PR in hammer-cli to better handle these translation issues. No pressure to review this since it's on the Phoenix board. I'll update this PR in a sec.

https://github.com/theforeman/hammer-cli/pull/397